python/vowpalwabbit/version.py

```diff
@@ -0,0 +1,5 @@
+# Provides the present version of VowpalWabbit
+
+import pkg_resources
+
+__version__ = pkg_resources.require('vowpalwabbit')[0].version
```

Comment:
Why do you need to get the 0th item here?

Reply:
It will produce the list whose first element is the instance of `'pkg_resources.Distribution'` of which `version` is an attribute created in `setup.py`